examples/firewall/networking_secgroup_rule/main-one-port.tf

```diff
@@ -0,0 +1,8 @@
+resource "vkcs_networking_secgroup_rule" "in_one_tcp" {
```

Comment:
copy this group from root firewall example

Reply:
used etcd